params/config.go

```diff
@@ -580,6 +583,13 @@ func (c *ChainConfig) GetMaxCodeSize(num *big.Int) int {
 			maxCodeSize = int(c.MaxCodeSize) * 1024
 		}
 	}
+	if c.Transitions != nil && len(c.Transitions) > 0 {
```

Comment:
the nil check is not useful, len(nil) will give 0
```suggestion
	if len(c.Transitions) > 0 {
```

Reply:
updated